oaknational/Oak-Web-Application
# chore(madge): add npm script to generate an svg graph of circular deps

## Description

- Adds cmd to generate a graph of circular dependencies
- Adds output dir (`madge/`) to gitignore

## How to test

1. Run `npm run madge:circulars`
2. Open file `madge/circulars.svg` 
4. You should see a diagram of the circular dependency files in the project

## Screenshots

<img width="1340" alt="image" src="https://github.com/oaknational/Oak-Web-Application/assets/12934669/70a9adec-501a-4c2a-9f12-cba3c9e95d6c">


## Checklist
